BUILD.md

```diff
@@ -59,9 +55,9 @@ Runs after the top-level `yarn install`. This ensures `./packages/plexus` builds
 
 Note that `./packages/plexus` does not yet have any tests, as tracked in issue [#340](https://github.com/jaegertracing/jaeger-ui/issues/340).
 
-`./packages/jaeger-ui` uses [Jest](https://jestjs.io/) for testing. It can be useful to directly run tests for that package by running `yarn test` from its directory, rather than the repository root.
+`./packages/jaeger-ui` uses [Jest](https://jestjs.io/) for testing. It can be useful to directly run tests for that package by running `npm run test` from its directory, rather than the repository root.
 
-Tests for React components in `./packages/jaeger-ui` make extensive use of Jest's [snapshot testing](https://jestjs.io/docs/28.x/snapshot-testing) functionality. These snapshots can be regenerated by running `yarn test -u` from the package directory to regenerate all snapshots, or `yarn test -u --testNamePattern <test name>` to regenerate snapshots for a subset of tests only.
+Tests for React components in `./packages/jaeger-ui` make extensive use of Jest's [snapshot testing](https://jestjs.io/docs/28.x/snapshot-testing) functionality. These snapshots can be regenerated by running `npm run --workspaces test -- -u` from the package directory to regenerate all snapshots, or `npm run --workspaces test -- -u --testNamePattern <test name>` to regenerate snapshots for a subset of tests only.
```

Comment:
The new commands are pretty unusable for something that we need to run often, like executing a single test. Is it possible to have some sort of shortcut? Why can't we use something like "npm test ..."?

Reply:
The command propagation changed, in yarn `yarn run somescript arg1` forwards `arg1` to `somescript`, but with npm you need to write `npm run somescript -- arg1`, so npm can differentiate between args for `npm` and args for the script.

I updated the workspaces invocation, now additional args are correctly passed to the `test` script in the root package.json, which then forwards it to the `test` script of all workspaces. The invocation now is almost identical.